packages/mobile-app/hooks/useHideBalances.ts

```diff
@@ -0,0 +1,18 @@
+import { useFacade } from "@/data/facades";
+import { useEffect, useState } from "react";
+
+/**
+ * Hook that provides the current hide balances setting
+ * @returns The current hide balances state
+ */
+export function useHideBalances(): boolean {
+  const facade = useFacade();
+  const [hideBalances, setHideBalances] = useState(false);
```

Comment:
Maybe should default to hidden here? I tested it and didn't notice this, but seems like potentially could allow for a flash of balance while the setting loads. Or maybe provide a loading state so you can skip rendering until the setting is loaded